v1.0/v1.0/bwa-mem-tool.cwl

```diff
@@ -5,8 +5,6 @@ cwlVersion: v1.0
 class: CommandLineTool
 
 hints:
-  - class: ResourceRequirement
-    coresMin: 4
```

Comment:
Can we replace this with other tests that provide minimal coverage of `ResourceRequirement`?